frontend/src/components/FullProductList.js

```diff
@@ -1,29 +1,47 @@
 import React, { useState } from 'react'
+import { SortDown, SortUpAlt } from 'react-bootstrap-icons'
 import useProducts from '../hooks/useProducts'
-import ProductTable from './ProductTable'
 import ListElement from './listOfProducts/ListElement'
+import ListHeader from './listOfProducts/ListHeader'
 import Searchbar from './Searchbar'
 
 const FullProductList = () => {
   const [searchKeyword, setSearchKeyword] = useState('')
-  const { products } = useProducts(searchKeyword)
+  const {
+    products,
+    sortKey,
+    setSortKey,
+    isSortAscending,
+    setAscendingSort,
+  } = useProducts(searchKeyword)
+
+  const onColumnClick = (value) => {
+    setAscendingSort(!isSortAscending)
+    setSortKey(value)
+  }
+
+  const SortingSign = (value) => {
```

Comment:
Értem, hogy eggyel feljebb emelted, de nem erre gondoltam, hanem hogy maga a SortingSign lehetne egy React komponens és a sortKey, isSortAscending, value meg prop-ként jönne be neki